### DISCUSSION TITLE
How to obtain all misorientation axes at LAGBs in the selected grain

### DISCUSSION THREAD

**Discussion by SingPr** [mtex-toolbox/mtex#1663] [Ask Anything]
Hi! all MTEXers,
     I want to plot all misorientation axes at the lower angle grain boundaries(LAGBs) with 1-2° in a selected **grain 149** like below: 
![149](https://user-images.githubusercontent.com/89964907/203720442-27f992fb-e6a3-4fe8-af59-60a6e33a42df.jpg)
The threshold values for grain boundaries are set at 1-15°. I I tried everything possible (the documentation https://mtex-toolbox.github.io/TiltAndTwistBoundaries.html and discussion #1426, #1370, ...). However, it doesn"t work. 
I tried to set the threshold value of the grain boundary to 1-2°, and then calculated the misorientation axis at LAGBs with 1-2°. but the original **grain 149** has been divided into several grains as follows:
![149_1](https://user-images.githubusercontent.com/89964907/203728337-28c828dc-973c-4379-a5b0-285d2437b657.jpg)
In this case, the misorientation axes within a single partite grain (such as **grain 1271**) can be obtained by using the method in the documentation https://mtex-toolbox.github.io/TiltAndTwistBoundaries.html:
![25](https://user-images.githubusercontent.com/89964907/203728928-4f5fe419-3254-411e-a689-a21d8b537f80.jpg)

 **But, how to obtain all misorientation axes at LAGBs with 1-2° in the original grain 149**?

If anyone understands the question or knows another function of MTEX that calculates the misorientation axis at the LAGBs with 1-2° in a selected grain, I will deeply appreciate any guidelines!

Thanks for helping
Sing

**Comment by kilir** [comment]
Hi,
I something like this may be what you are looking for:

```
% grain of interest
goi = grains('id',65);

%all inner boudnaries of goi
goiIB = goi.innerBoundary;

%condition for angular range (2nd one needed unless identical to the one 
% supplied to calcGrains)
cond = goiIB.misorientation.angle < 2 * degree & ...
       goiIB.misorientation.angle > 1.5 *degree;

plot(goi)
hold on
plot(goiIB(cond),'linecolor','r')
hold off

% and these are the misorientations axes
goiIB(cond).miso
```

Cheers,
Rüdiger

**Reply by SingPr** [reply]
Thanks, Rüdiger, it's work.
Sing
